backend/core/urls.py

```
@@ -49,6 +49,11 @@
     ComplianceAssessmentViewSet,
     basename="compliance-assessments",
 )
+router.register(
+    r"campaigns",
+    CampaignViewSet,
```

Ruff (0.11.9)

`CampaignViewSet` may be undefined, or defined from star imports

(F405)